packages/common/src/utils/abort-signal-utils.ts

```diff
@@ -62,6 +63,7 @@ export async function abortable<T>(
   }
   original.addEventListener('abort', onAbort)
   if (original.aborted) controller.abort()
+  setMaxListeners(Infinity, controller.signal)
```

Comment:
It seems like this might open us up to possibly memory leaks. Can we make the value passed in and configurable?

Alternate is actually to provide a metric somehow to make sure this isn't constantly growing, but not sure if there's an easy way to do that.